ofrak_core/ofrak/core/comments.py

```diff
@@ -82,10 +97,24 @@ async def modify(self, resource: Resource, config: DeleteCommentModifierConfig)
         except NotFoundError:
             comments = {}
         try:
-            del comments[config.comment_range]
+            if len(config.comment_range) == 1:
+                config.comment_range = (config.comment_range[0], None)
+
+            if config.comment_range[1] is None:
```

Comment:
See comment above about having a separate `config.comment_text` and `config.comment_range` to clean up this section